eclipsePlugin/build.gradle

```diff
@@ -34,6 +36,12 @@ def eclipsePluginId = 'com.github.spotbugs.plugin.eclipse'
 def localProps = new Properties()
 localProps.load(new FileInputStream("$projectDir/local.properties"))
 
+ext {
+  eclipseRootDir = localProps.getProperty('eclipseRoot.dir')
+  eclipseExecutable = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/MacOS/eclipse" : "/eclipse")
+  eclipsePluginsDir = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/Eclipse/plugins" : "/plugins")
+}
```

Comment:
SpotBugs Eclipse Plugin supports legacy Eclipse such as 3.6.2, so the condition to switch directory should be more complex. For instance, `eclipsePluginsDir ` for Eclipse 3.6.2 for OS X is not `${eclipseRootDir}/Contents/Eclipse/plugins` but `${eclipseRootDir}/plugins`.

I guess it's better to check not OS but existence of files like below:

```groovy
eclipseExecutable = eclipseRootDir + '/eclipse'
if (!new File(eclipseExecutable).isFile()) {
  eclipseExecutable = eclipseRootDir + '/Contents/MacOS/eclipse'
}
```